externs/shaka/player.js

```diff
@@ -968,6 +969,10 @@ shaka.extern.ManifestConfiguration;
  * @description
  * The StreamingEngine's configuration options.
  *
+ * @property {shaka.config.CodecSwitchingStrategy} codecSwitchingStrategy
+ *  Allow codec switching strategy. Default behavior is disabled.
```

Comment:
This currently only applies to audio, right? You should mention that.

Reply:
Video support was added in the latest commit.